sdk/cosmos/azure-cosmos/src/test/java/com/azure/cosmos/CosmosClientBuilderTest.java

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+package com.azure.cosmos;
+
+import com.azure.cosmos.implementation.TestConfigurations;
+import com.google.common.collect.ImmutableList;
+import org.testng.annotations.Test;
+
+import java.net.URISyntaxException;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class CosmosClientBuilderTest {
```

Comment:
Thanks for adding unit test coverage for this, can you please also add a couple of happy path scenarios in this file, since we updated the `validateConfig()` API in `CosmosClientBuilder`. 
It will ensure there is no regression from this change.

Reply:
unfortunately the happy path scenarios can't come here. because require a real endpoint. the instantiation of the client without a real prod endpoint will fail.

Happy path scenarios are tested elsewhere in our end to end tests when we run prod CI.

Reply:
Sounds good @moderakh  - can we run the full CI for this PR to make sure everything is good on end to end tests ?